blockly-rtc/src/Location.js

```diff
@@ -16,96 +16,64 @@
  */
 
 /**
- * @fileoverview Object for communicating an update to a Marker's location.
+ * @fileoverview Object representing a user's location.
```

Comment:
location is a little ambiguous as it could mean perhaps a user's physical location. Can we make that clear here? Perhaps, ``Object representing a user's cursor location``?

Reply:
Discussed offline: Location > Position and a more descriptive file overview.